src/Components/Programs/styles.js

```diff
@@ -58,12 +58,22 @@ export const ArrowNavigation = styled(View,{
     marginTop:20,
     marginBottom:8,
     width: '80%',
-    justifyContent:'space-between'
+    justifyContent:'space-between',
+    cursor: 'pointer',
+    color: '#42aaf5'
 })
 export const ArrowText = styled(Text,{
     fontSize:16,
     marginLeft:8,
-    marginRight:8
+    marginRight:8,
+    cursor: 'pointer',
+    color: '#42aaf5',
+})
+export const ArrowLeftOutlined = styled(View,{
+    color: '#42aaf5',
+})
+export const ArrowRightOutlined = styled(View,{
+    color: '#42aaf5',
```

Comment:
Can't we have only one of those as both are same
```suggestion
export const ArrowOutlined = styled(View,{
    color: '#42aaf5',
})
```